package.json

```diff
@@ -58,15 +59,17 @@
     "marked": "0.7.0",
     "mocha": "5.2.0",
     "node-libs-browser": "2.1.0",
+    "postcss": "^8.4.4",
     "postcss-easing-gradients": "3.0.1",
-    "postcss-loader": "3.0.0",
-    "postcss-safe-parser": "3.0.1",
+    "postcss-less": "^5.0.0",
+    "postcss-loader": "^4.0.0",
```

Comment:
had to update postcss-loader to `v4` even though latest is `v6` because versions greater than 5 require webpack 5.